pkgs/applications/editors/jetbrains/plugins/default.nix

```
@@ -89,14 +89,15 @@ rec {
       passthru.plugins = plugins ++ (ide.plugins or [ ]);
       newPlugins = plugins;
       disallowedReferences = [ ide ];
-      nativeBuildInputs = [ autoPatchelfHook ] ++ (ide.nativeBuildInputs or [ ]);
+      nativeBuildInputs = (if !stdenv.isDarwin then [ autoPatchelfHook ] else []) ++ (ide.nativeBuildInputs or [ ]);
```

This can also be simplified to
```suggestion
      nativeBuildInputs = (lib.optional stdenv.isLinux autoPatchelfHook) ++ (ide.nativeBuildInputs or [ ]);
```